### PR TITLE
Local permissions test

### PR DESCRIPTION
**Pull request by chenilim**
#### Summary
Added local (Personal Server) version of permissions_test.
* Added missing permission check for duplicating board to another team
* Refactored permissions_test to handle variable user IDs (for local server tests)

Diff `permissinos_tests.go` with `local_permissions_tests.go` to see differences in expected responses.

### CODE REVIEW

File: server/integrationtests/local_permissions_test.go [hunk 1]
```diff
@@ -0,0 +1,2047 @@
...
+}
+
+func TestLocalPermissionsRegister(t *testing.T) {
+	th := SetupTestHelperLocalMode(t)
+	defer th.TearDown()
+	clients := setupLocalClients(th)
+	testData := setupData(t, th)
+
+	ttCases := []TestCase{
+		{"/register", methodPost, "", userAnon, http.StatusInternalServerError, 0},
```
Comment (jespino) [inline server/integrationtests/local_permissions_test.go:1592]: Why register method is generating an `InternalServerError`?

**Review by jespino — CHANGES_REQUESTED** [review]
Everything looks good to me, except the InternalServerError return on register.

File: server/integrationtests/local_permissions_test.go (hunk 1 above)
Comment (chenilim) [reply server/integrationtests/local_permissions_test.go]: Thanks. I've updated the test case to make a successful register call.

**Review by jespino — APPROVED** [review]
LGTM

**Pull request merged**